hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/table/ITTestHoodieDataSource.java

```diff
@@ -1553,14 +1553,14 @@ void testBuiltinFunctionWithCatalog(String operation) {
   }
 
   @ParameterizedTest
-  @EnumSource(value = HoodieTableType.class)
-  void testWriteAndReadWithDataSkipping(HoodieTableType tableType) {
+  @MethodSource("tableTypeAndDataSkippingParams")
+  void testWriteAndReadWithDataSkipping(HoodieTableType tableType, boolean dataSkipping) {
     TableEnvironment tableEnv = batchTableEnv;
     String hoodieTableDDL = sql("t1")
```

Comment:
Can we write a new test instead of modifying existing one.

Reply:
@danny0405, I have added a new test to verify.